src/Disks/DiskLocalCheckThread.cpp

```diff
@@ -24,26 +32,36 @@ void DiskLocalCheckThread::startup()
     need_stop = false;
     retry = 0;
     task->activateAndSchedule();
+    LOG_INFO(
+        log,
+        "Disk check started with period {}",
+        formatReadableTime(static_cast<double>(check_period_ms) * 1e6 /* ns */));
+}
+
+int diskStatusChange(bool old_val, bool new_val)
+{
+    return static_cast<int>(new_val) - static_cast<int>(old_val);
 }
 
 void DiskLocalCheckThread::run()
 {
     if (need_stop)
         return;
-
+    bool readonly = disk->isReadOnly();
+    bool broken = disk->isBroken();
     bool can_read = disk->canRead();
     bool can_write = disk->canWrite();
     if (can_read)
     {
         if (disk->broken)
-            LOG_INFO(log, "Disk {0} seems to be fine. It can be recovered using `SYSTEM RESTART DISK {0}`", disk->getName());
```

Comment:
Let's log the disk name. Is there a specific reason why it's being removed?
Same comment for other logging statements.